schema.sql

```diff
@@ -0,0 +1,122 @@
+CREATE TABLE Employees (
+    eid INTEGER,
+    did INTEGER,
+    ename TEXT,
+    email TEXT UNIQUE,
+    resign_date DATE,
+    contact1 INTEGER,
+    contact2 INTEGER,
+    contact3 INTEGER,
```

Comment:
Would it be better to rename these `home_number`, `office_number`, `mobile_number`?